tests/phpunit/tests/link/postNavigationLinkAttributeFilters.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+/**
+ * Test post navigation link attribute filters.
+ * @ticket 55751
+ * @since 6.2.0
+ *
+ * @covers ::next_posts_link_attributes
+ * @covers ::previous_posts_link_attributes
```

Comment:
```suggestion
 * Test post navigation link attribute filters.
 *
 * @since 6.2.0
 * @ticket 55751
 *
 * @group link
 *
 * @covers ::next_posts_link_attributes
 * @covers ::previous_posts_link_attributes
```

Per [56070](https://core.trac.wordpress.org/ticket/56070), 
- `@since` should precede `@ticket`
- There should be a line break between `Summary` and `@ticket`
- The test should grouped to `link`